app-android-xamarin/Moodis/Moodis/Feature/CameraFeature/CameraFragment.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.Content.PM;
+using Android.Hardware;
+using Android.OS;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+
+
+namespace Moodis.Feature.CameraFeature
+{
+    public class CameraFragment : Fragment
+    {
+        private readonly string TAG = nameof(CameraFragment);
+        Camera camera;
```

Comment:
Isn't `Android.Hardware.Camera` deprecated? `Android.Hardware.Camera2` should be used instead, although if switching to newer camera lib would require a huge refactor of code, then maybe it is not worth the effort

Reply:
Camera2 works from android 5.2, and is also mostly used for professional photography(editing photos, sharpening etc.. .). However as you said it's an overly complicated api. (just to open the camera you need ~1000 lines). As a lot of apps still use Android.Hardware.Camera and it is a lot easier to use, it's support won't be fully dropped any time soon.